ext/curl/interface.c

```diff
@@ -1896,7 +1896,7 @@ PHP_FUNCTION(curl_init)
 		}
 	}
 
-	ZEND_REGISTER_RESOURCE(return_value, ch, le_curl);
+	ZVAL_RES(return_value, zend_register_resource(ch, le_curl));
```

Comment:
since if we keep the marco, it probably confused the people, since the implmenation of macro is changed...